.github/ISSUE_TEMPLATE/other.yaml

```diff
@@ -0,0 +1,41 @@
+name: Other
+description: Use this for any other issues. Please do NOT create blank issues
+title: '[OTHER] write a small description here'
+body:
+- type: textarea
+  id: issuedescription
+  attributes:
+    label: What would you like to share?
+    description: Provide a clear and concise explanation of your issue.
+  validations:
+    required: true
+
+- type: dropdown
+  id: browser
+  attributes:
+    label: '✨ Browser'
+    description: 'What browser are you using ?'
```

Comment:
```suggestion
    description: 'What browser are you using?'
```

Reply:
Irrelevant after 478be206f70ce90d34cf2179c442cc48b906b327 as we do not ask about the browser anymore.